stream_test.go

```diff
@@ -220,6 +220,35 @@ func TestSetRowNilValues(t *testing.T) {
 	assert.NotEqual(t, ws.SheetData.Row[0].C[0].XMLName.Local, "c")
 }
 
+func TestSetRowWithStyle(t *testing.T) {
```

Comment:
I suggest that using the `TestStream` prefix for the stream test functions, so using the `TestStreamSetRowWithStyle` instead of `TestSetRowWithStyle`, the existing test function names `TestSetRow`, `TestSetRowNilValues` and `TestSetCellValFunc` also should be rename.